src/common/decorators/auth-with-roles.decorator.ts

```diff
@@ -0,0 +1,10 @@
+import { UseGuards, applyDecorators } from '@nestjs/common';
+
+import { JwtAuthGuard } from 'src/modules/auth/guard/jwt-auth.guard';
```

Comment:
please, configure absolute paths so that they look like 
```suggestion
import { JwtAuthGuard } from 'modules/auth/guard/jwt-auth.guard';
```